ssl/quic/quic_channel.c

```diff
@@ -480,6 +484,37 @@ ossl_quic_channel_get_terminate_cause(const QUIC_CHANNEL *ch)
     return ossl_quic_channel_is_term_any(ch) ? &ch->terminate_cause : NULL;
 }
 
+int ossl_quic_channel_can_send_closing_bytes(QUIC_CHANNEL *ch, size_t n)
```

Comment:
This does not seem to be called from anywhere.

Reply:
Lost somewhere. It was meant to be called from the TXP.
Anyway, refactored now so it lives in the TXP.